### PR TITLE
Feature/backend models seba

### PR DESCRIPTION
**Pull request by sebastianmerced**
This branch contains the models of data point and data log as well as its serializers and views.

### CODE REVIEW

File: backend/rumergy_backend/rumergy/serializers/data_log_serializer.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from rest_framework import serializers
+from rumergy_backend.rumergy.models import DataLog
+from django.contrib.auth.models import User
+
+class DataLogSerializer(serializers.ModelSerializer):
+    
+    # meter = serializers.PrimaryKeyRelatedField(many=True, queryset=Meter.objects.all())
+    user = serializers.PrimaryKeyRelatedField(many=False, queryset=User.objects.all())
+    # TODO: Import Meter model
+
```
Comment (milton-pagan) [inline backend/rumergy_backend/rumergy/serializers/data_log_serializer.py:10]: Should data log measures be included here?

File: backend/rumergy_backend/rumergy/serializers/data_point_serializer.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+from rest_framework import serializers
+from rumergy_backend.rumergy.models.data_point import DataPoint
+# from rumergy_backend.rumergy.models.model import Model
+
+class DataPointSerializer(serializers.ModelSerializer):
+    
+    # model = serializers.PrimaryKeyRelatedField(queryset=Model.objects.all())
```
Comment (milton-pagan) [inline backend/rumergy_backend/rumergy/serializers/data_point_serializer.py:7]: Should be `many=False` I think

**Review by milton-pagan — CHANGES_REQUESTED** [review]
Left some suggestions

File: backend/rumergy_backend/rumergy/serializers/data_log_serializer.py (hunk 1 above)
Comment (Orlando-RL) [reply backend/rumergy_backend/rumergy/serializers/data_log_serializer.py:10]: Yeah, it should

File: backend/rumergy_backend/rumergy/serializers/data_point_serializer.py (hunk 2 above)
Comment (Orlando-RL) [reply backend/rumergy_backend/rumergy/serializers/data_point_serializer.py:7]: Model relationship doesn't go here, goes in model's serializer

**Review by milton-pagan — APPROVED** [review]

**Pull request merged**